backend/user.js

```diff
@@ -166,6 +166,23 @@ router.post('/myprofile', authenticateToken, async (req, res) => {
         physicianId: physician.id
       }
     });
+
+    // Create a new notification for the physician
+    const newNotification = await prisma.notification.create({
+      data: {
+        content: `New patient profile created: ${firstname} ${lastname}`,
+        physicianId: physician.id,
+      },
+    });
+
+    // Send notification via WebSocket
+    if (userToWS[physician.userId]) {
```

Comment:
nice! does this piece of code handle when a user isn't connected?

Reply:
No, that piece checks if a physician is connected to a Websocket at that the time the notification is to be sent.